frontend/src/App.js

```diff
@@ -23,8 +23,21 @@ import SideNav, { NavItem, NavIcon } from '@trendmicro/react-sidenav';
 import '@trendmicro/react-sidenav/dist/react-sidenav.css';
 import Center from 'react-center';
 
+import Card from './routes/Components/Card'
+
+const logo = {
+  width: '3.125rem', 
+  paddingTop: '0.625rem', 
+  paddingBottom: '0.3125rem'
+}
+
+const paddingBottom={
```

Comment:
this class name does not tell me what elements is used for.. even like "spacer" is a better name. just dont name it based on the attributes cuz what if u want to add more/this is just bad practice?

Reply:
Changed it to `logoutSpacer` it's just a spacer between the logout button and the rest of the buttons in the navbar.